ethereum/core/src/test/java/org/hyperledger/besu/ethereum/trie/diffbased/common/storage/flat/FlatDbStrategyProviderTest.java

```diff
@@ -43,8 +45,9 @@
 
 @ExtendWith(MockitoExtension.class)
 class FlatDbStrategyProviderTest {
```

Comment:
I think this class should be renamed too right? to `BonsaiFlatDbStrategyProviderTest` as you're only testing the Bonsai one

Reply:
good catch I also need to move it